doc/onboarding-extras.md

```diff
@@ -31,7 +31,7 @@
 | `test/*`                                 | @nodejs/testing                                                       |
 | `tools/node_modules/eslint`, `.eslintrc` | @not-an-aardvark, @silverwind, @trott                                 |
 | build                                    | @nodejs/build                                                         |
-| ES Modules                               | @bmeck, @Fishrock123, @guybedford, @MylesBorins, @targos              |
+| ES Modules                               | @nodejs/modules                                                       |
```

Comment:
If we are talking about all things module...maybe this can be more specific, e.g. putting it down as `lib/{internal/}module.js` and `src/module_wrap.*`?